packages/react/src/components/Auth/Auth.tsx

```diff
@@ -173,7 +174,7 @@ function Auth({
             appearance={appearance}
             supabaseClient={supabaseClient}
             setAuthView={setAuthView}
-            redirectTo={redirectTo}
+            redirectTo={resetPasswordRedirectTo ?? redirectTo}
```

Comment:
Shouldn't this be designed to specify the redirect URL based on the `view` state of the Auth component?

```js
redirectTo={view === 'forgotten_password' ? resetPasswordRedirectTo ?? redirectTo : redirectTo}
```

Reply:
Ah, I now see that this is already under the `case VIEWS.FORGOTTEN_PASSWORD:` condition, thus there's no need to check `view`...  Disregard.